pypixel.py

```diff
@@ -261,4 +261,4 @@ def main(self, action, args={}):
 				loaded = self.api.main(action, args)
 				if "throttle" not in loaded:
 					return loaded
-		return loaded
+		return loaded
```

Comment:
fix this >.>

also this should be designed to work independent of version, if you're updating it.

I don't think we'll merge it otherwise.